.github/workflows/ci.yml

```diff
@@ -34,8 +34,10 @@ jobs:
     steps:
       - name: Checkout Repo
         uses: actions/checkout@v4
-      - uses: mamba-org/setup-micromamba@v2
```

Comment:
We can use v2 here right?

Reply:
IMO we can't: `v2` failed with: https://github.com/pydata/sparse/actions/runs/11685373919/job/32538684569#step:3:460 which is explained in https://github.com/mamba-org/setup-micromamba/issues/227 (also linked it in the file). Versions used here (for now until v2 is fixed) solve it.

Reply:
I don't know why an issue reported in September appeared in our CI only now but I will go ahead and merge it with `v1.9`.

Reply:
I will go ahead and merge it now. I'll open a separate PR with `v2` to track when it's fixed upstream.

Reply:
Here's a PR that reproduces it: https://github.com/pydata/sparse/pull/805